### DISCUSSION TITLE
403 (Forbidden): Your user does not have permission to define specific fields.

### DISCUSSION THREAD

**Discussion by di2mot** [censys/censys-python#536] [Q&A] [answered]
Hi there,
In the API documentation I see that I can specify which fields I want to return in the query. But when I try to do it I get this error. I have not found where I can see the list of restrictions and what is available in the basic version.

Can you please tell me where I can see the restrictions?

**Comment by thehappydinoa** [comment, marked as answer]
Hey there,

This is a great question, currently field specific search and regex searching are the only two restrictions. For additional context please see this issue: https://github.com/censys/censys-python/issues/524. I will let the team know about this question for out documentation. Thanks!

**Reply by di2mot** [reply]
Thanks)
